layer/store/wallet/index.ts

```diff
@@ -1,32 +1,47 @@
 import { defineStore } from 'pinia'
 import {
+  Msgs,
+  msgsOrMsgExecMsgs,
   getEthereumAddress,
   getInjectiveAddress,
   getDefaultSubaccountId
 } from '@injectivelabs/sdk-ts'
+import { GeneralException } from '@injectivelabs/exceptions'
+import { StatusType } from '@injectivelabs/utils'
 import {
   Wallet,
   isEthWallet,
   isCosmosWallet,
   isCosmosBrowserWallet
 } from '@injectivelabs/wallet-ts'
-import { StatusType } from '@injectivelabs/utils'
 import {
   validateCosmosWallet,
   confirmCorrectKeplrAddress
-} from './../wallet/cosmos'
+} from './../../wallet/cosmos'
+import {
+  validateOkxWallet,
+  isOkxWalletInstalled
+} from './../../wallet/okx-wallet'
 import {
   validateTrustWallet,
   isTrustWalletInstalled
-} from './../wallet/trust-wallet'
-import { IS_DEVNET } from './../utils/constant'
-import { getAddresses } from './../wallet/wallet'
-import { walletStrategy } from './../wallet/wallet-strategy'
-import { isBitGetInstalled, validateBitGet } from './../wallet/bitget'
-import { validatePhantom, isPhantomInstalled } from './../wallet/phantom'
-import { validateMetamask, isMetamaskInstalled } from './../wallet/metamask'
-import { validateOkxWallet, isOkxWalletInstalled } from './../wallet/okx-wallet'
-import { EventBus, WalletConnectStatus } from './../types'
+} from './../../wallet/trust-wallet'
+import { IS_DEVNET } from './../../utils/constant'
+import { getAddresses } from './../../wallet/wallet'
+import { walletStrategy } from './../../wallet/wallet-strategy'
+import { isBitGetInstalled, validateBitGet } from './../../wallet/bitget'
+import { validatePhantom, isPhantomInstalled } from './../../wallet/phantom'
+import { validateMetamask, isMetamaskInstalled } from './../../wallet/metamask'
+import { EventBus, GrantDirection, WalletConnectStatus } from './../../types'
+import { AutoSign } from '@/types/wallet'
```

Comment:
Lets not use `@/` in layer, it will fail on render.

Also we should be able to import this directly from `types`

Reply:
Also we should put types at the bottom